httpgrpc/server/server.go

```diff
@@ -32,17 +32,31 @@ var (
 	DoNotLogErrorHeaderKey = http.CanonicalHeaderKey("X-DoNotLogError")
 )
 
+type Options func(*Server)
+
+var (
+	Return4XXErrorsOption Options = func(s *Server) {
```

Comment:
[nit] A couple of nits:
- Looks a cumbersome way of defining a function
- It's commong to start such functions with `With...` and not having the `Option` suffix

So my suggestion is:

```suggestion
func WithReturnErrorOn4XX(s *Server) {
```

Reply:
I will implement this recommendation. Thank you